moveit_plugins/moveit_fake_controller_manager/src/moveit_fake_controller_manager.cpp

```diff
@@ -262,12 +268,9 @@ class MoveItFakeControllerManager : public moveit_controller_manager::MoveItCont
    * Controllers are all active and default.
    */
   moveit_controller_manager::MoveItControllerManager::ControllerState
-  getControllerState(const std::string& /*name*/) override
```

Comment:
Shouldn't the same be implemented for the SimpleControllerManager?

Reply:
Probably. @llach, could you provide the same implementation for SimpleControllerManager as well?
I suggest to split this PR into two separate commits then:
1. forwarding the controller set
2. fixing FakeControllerManager and SimpleControllerManager to consider the config `default`
So, Luca, please cleanup the commit history by appropriately squashing your changes.